### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by capablemonkey**

### CODE REVIEW

File: omniauth-greenhouse.gemspec [hunk 1]
```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w(lib)
 
   spec.add_development_dependency 'bundler', '~> 1.3'
-  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rake', '~> 0'
```
Comment (tfrey7) [inline omniauth-greenhouse.gemspec]: Don't think this one actually matters.

File: omniauth-greenhouse.gemspec [hunk 2]
```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
...
 
-  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1.1.1'
-  spec.add_runtime_dependency 'omniauth', '~> 1.2.1'
+  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1'
+  spec.add_runtime_dependency 'omniauth', '~> 1.2.2'
```
Comment (tfrey7) [inline omniauth-greenhouse.gemspec]: We can actually remove this guy I think.  the `omniauth-oauth2` gem includes this dependency:

https://rubygems.org/gems/omniauth-oauth2/versions/1.4.0

Comment (capablemonkey) [inline omniauth-greenhouse.gemspec]: The issue we're trying to fix is fixed in `omniauth` 1.3.1, but `omniauth-oauth2`'s latest release (1.4) has a minor version requirement on `omniauth` ~> 1.2.  There is an [outstanding issue](https://github.com/intridea/omniauth-oauth2/issues/87) to upgrade to 1.3.  I think we should add:

```
spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```

to ensure that 1.3.1 is being used.

File: omniauth-greenhouse.gemspec [hunk 3]
```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rake'
 
-  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1.1.1'
-  spec.add_runtime_dependency 'omniauth', '~> 1.2.1'
+  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1'
+  spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```
Comment (tfrey7) [inline omniauth-greenhouse.gemspec]: Do we need this line anymore?  I'm pretty sure it's already a dependency of `omniauth-oauth2`

**Comment by tfrey7** [comment]
Looks good otherwise.

File: omniauth-greenhouse.gemspec (hunk 3 above)
Comment (capablemonkey) [inline omniauth-greenhouse.gemspec]: @tfrey7 The issue we're trying to fix is fixed in `omniauth` 1.3.1, but `omniauth-oauth2`'s latest release (1.4) has a minor version requirement on `omniauth` ~> 1.2.  There is an [outstanding issue](https://github.com/intridea/omniauth-oauth2/issues/87) to upgrade to 1.3.  I think we should add:

```
spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```

to ensure that 1.3.1 is being used.

**Comment by capablemonkey** [comment]
Thanks @tfrey7!

**Pull request merged**